src/rust/demikernel/libos/network/mod.rs

```diff
@@ -13,13 +13,13 @@ pub mod queue;
 // Imports
 //======================================================================================================================
 
+#[cfg(all(feature = "catnap-libos"))]
```

Comment:
I believe this will cause a warning on all of the other libOSes that have not moved yet

Reply:
Added this condition to avoid warning on other LibOSes.